08/26/2020 11:16:24 PM - filter_responses.py - INFO - Execution begins {current_dir:/scripts/aida/python, script_name:filter_responses.py, arguments:--log /score/logs/filter-responses.log /scripts/aida/python/input/aux_data/log_specifications.txt /AUX-data/AIDA_Annotation_Ontology_Phase2_V1.1_typemappings.tab /AUX-data/AIDA_Annotation_Ontology_Phase2_V1.1_slotnamemappings.tab /AUX-data/encoding_modality.txt /AUX-data/LDC2020E11.coredocs-13.txt /AUX-data/LDC2020E11.parent_children.tsv /AUX-data/LDC2020E11.sentence_boundaries.txt /AUX-data/LDC2020E11.image_boundaries.txt /AUX-data/LDC2020E11.keyframe_boundaries.txt /AUX-data/LDC2020E11.video_boundaries.txt /AUX-data/LDC2020E11.annotated_regions.txt example-run /score/SPARQL-VALID-output /score/SPARQL-FILTERED-output}
08/26/2020 11:16:26 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20874' has more than two fillers at /score/SPARQL-VALID-output/KC003A6NA.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:74
08/26/2020 11:16:26 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20874' has more than two fillers at /score/SPARQL-VALID-output/KC003A6NA.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:75
08/26/2020 11:16:27 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20001' has more than two fillers at /score/SPARQL-VALID-output/IC001V8WF.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:40
08/26/2020 11:16:27 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20001' has more than two fillers at /score/SPARQL-VALID-output/IC001V8WF.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:41
08/26/2020 11:16:27 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20001' has more than two fillers at /score/SPARQL-VALID-output/IC001V8WF.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:42
08/26/2020 11:16:27 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20001' has more than two fillers at /score/SPARQL-VALID-output/IC001V8WF.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:43
08/26/2020 11:16:27 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20002' has more than two fillers at /score/SPARQL-VALID-output/IC001V8WF.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:46
08/26/2020 11:16:27 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20002' has more than two fillers at /score/SPARQL-VALID-output/IC001V8WF.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:47
08/26/2020 11:16:27 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20378' has more than two fillers at /score/SPARQL-VALID-output/IC001VGO2.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:54
08/26/2020 11:16:27 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20378' has more than two fillers at /score/SPARQL-VALID-output/IC001VGO2.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:55
08/26/2020 11:16:27 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20378' has more than two fillers at /score/SPARQL-VALID-output/IC001VGO2.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:56
08/26/2020 11:16:27 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20378' has more than two fillers at /score/SPARQL-VALID-output/IC001VGO2.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:57
08/26/2020 11:16:27 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20378' has more than two fillers at /score/SPARQL-VALID-output/IC001VGO2.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:58
08/26/2020 11:16:27 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20378' has more than two fillers at /score/SPARQL-VALID-output/IC001VGO2.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:59
08/26/2020 11:16:27 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20378' has more than two fillers at /score/SPARQL-VALID-output/IC001VGO2.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:60
08/26/2020 11:16:27 PM - filter_responses.py - ERROR - Relation 'cluster-NILR20390' has more than two fillers at /score/SPARQL-VALID-output/IC001VGO2.ttl/AIDA_P2_TA1_AM_A0001.rq.tsv:88
